crates/sui-framework/sources/governance/genesis.move

```diff
@@ -104,11 +103,11 @@ module sui::genesis {
                 primary_address,
                 worker_address,
                 // Initialize all validators with uniform stake taken from the subsidy fund.
-                balance::split(&mut subsidy_fund, initial_validator_stake_mist),
+                option::some(balance::split(&mut subsidy_fund, initial_validator_stake_mist)),
                 option::none(),
                 gas_price,
                 commission_rate,
-                0, // start operating right away at epoch 0
+                true, // validator is active right away
```

Comment:
Would like your review on the special casing we are doing here for genesis validators @bmwill. Genesis validators' staking pools are activated right away with some initial stake, while validators joining after genesis will have to start with empty preactive pools and follow the steps to activate their pools.

Reply:
sounds good to me